src/pkg/cli/client/byoc/aws/byoc.go

```diff
@@ -51,9 +51,8 @@ var (
 type ByocAws struct {
 	*byoc.ByocBaseClient
 
-	cdImageTag      string
-	delegationSetId string
-	driver          *cfn.AwsEcs
+	cdImageTag string
```

Comment:
let's remove this state

Reply:
Done, also removed the lingering reference of it in DO